src/test/Logger.h

```diff
@@ -19,31 +19,12 @@
 #ifndef DRMOCK_SRC_TEST_LOGGER_H
 #define DRMOCK_SRC_TEST_LOGGER_H
 
-#include <mutex>
+#include "Utility.h"
```

Comment:
```suggestion
#include "../utility/Logger.h"
```
Not sure if it's a good idea to include master headers in project headers.